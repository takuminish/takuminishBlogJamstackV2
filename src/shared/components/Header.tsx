import {
  AppBar,
  Box,
  Container,
  SvgIcon,
  Toolbar,
  Typography,
} from "@mui/material";
import { Link } from "gatsby";
import React from "react";
import Logo from "../../assets/svg/Logo.svg";

function Header() {
  return (
    <AppBar
      position="static"
      sx={{
        backgroundColor: "#ffffff",
        boxShadow: "none",
        borderBottom: "1px solid #dddddd",
      }}
    >
      <Container maxWidth="xl">
        <Toolbar disableGutters>
          <Box>
            <Logo />
          </Box>
          <Box
            sx={{
              display: "flex",
              justifyContent: "start",
              gap: "10px",
            }}
          >
            <Typography
              variant="h5"
              component={Link}
              to="/"
              sx={{
                fontWeight: 700,
                color: "#6a7281",
                textDecoration: "none",
              }}
            >
              Takuminish.dev
            </Typography>
            <Typography
              variant="h6"
              component={Link}
              to="/blogs"
              sx={{
                fontWeight: 700,
                color: "#6a7281",
                textDecoration: "none",
              }}
            >
              Blogs
            </Typography>
          </Box>
        </Toolbar>
      </Container>
    </AppBar>
  );
}

export default Header;
