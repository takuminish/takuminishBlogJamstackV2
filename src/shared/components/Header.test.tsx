import React from "react";
import { render } from "@testing-library/react";
import Header from "./Header";

const createDOM = () => render(<Header />);

test("snapshot", () => {
  const { container } = createDOM();

  expect(container).toMatchSnapshot();
});
