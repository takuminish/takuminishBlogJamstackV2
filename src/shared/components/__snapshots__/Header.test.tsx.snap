// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-kql80c-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular css-r6ewbb-MuiToolbar-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <test-file-stub />
        </div>
        <div
          class="MuiBox-root css-14j8yx0"
        >
          <a
            aria-current="page"
            class="MuiTypography-root MuiTypography-h5 css-ewn5el-MuiTypography-root"
            href="/"
          >
            Takuminish.dev
          </a>
          <a
            class="MuiTypography-root MuiTypography-h6 css-1au34hm-MuiTypography-root"
            href="/blogs"
          >
            Blogs
          </a>
        </div>
      </div>
    </div>
  </header>
</div>
`;
